server/routes/todo.js

```diff
@@ -9,41 +9,49 @@ export default function (router) {
      * Returns all of the todos from the database.
      */
     router.get("/todos", async (req, res) => {
-        pool.query('SELECT * FROM todos', (err, result) => {
-            if (err) {
-                throw err;
-            }
 
+        const selectAllCommand = "SELECT * FROM todos";
```

Comment:
This is a nitpick, but technically it's not a command. It's a query. I would amybe changed the variable to `selectAllQuery` or more simply: `query`.

Reply:
Renamed all complicated variable commands to simpler "query"